packages/constants/src/constants.ts

```diff
@@ -17,7 +17,15 @@ export enum EnumBuildingTypes {
   MULTI_DWELLING = "multi-dwelling",
 }
 
+export enum EnumBreadcrumbIds {
```

Comment:
@stechkev I'm not necessarily against this, but I'm not sure we need it. This just duplicates the other strings we have for page titles. In the unit tests I'm using the helper function of getting the page name by using the URL segment. And the url segments are just the building type enums.

```javascript
GET_TESTID_BREADCRUMBS_BREADCRUMB(GET_PAGE_NAME(EnumBuildingTypes.MULTI_DWELLING))
```

Reply:
this is our enums for building type:
export enum EnumBuildingTypes {
  SINGLE_DWELLING = "single-dwelling",
  MULTI_DWELLING = "multi-dwelling",
}
and this is the one i made
export enum EnumBreadcrumbIds {
  HOME = "Home",
  MULTI_DWELLING_UNIT = "Multi-Dwelling Unit",
  SINGLE_DWELLING_UNIT = "Single-Dwelling Unit",
}
so i can just string builder it up, but it felt like it would be a good idea to use an enum so that when we go ahead and add more pages we could continue to add to that list. alternatively, i'd be happy if we changed the test id from data-testid="breadcrumb-Multi-Dwelling Unit" to data-testid="breadcrumb-multi-dwelling" which would let me use the existing EnumBuildingTypes without having to do any string transformations. 

The other issue is HOME, which i think i just have to hardcode somewhere as a string unless you have a better idea. if we get rid of the enum we can just have that HOME string in the test-ids like we do with breadcrumb-Last, or i can just have it hardcoded in the test, whichever is easier